pkg/apis/backupdriver/v1/snapshot.go

```diff
@@ -155,12 +155,12 @@ const (
 )
 
 type CloneStatus struct {
-	Phase ClonePhase `json:"phase"`
+	Phase ClonePhase `json:"phase,omitempty"`
```

Comment:
Add // +optional

Reply:
This should also be *ClonePhase if it is optional

Reply:
ClonePhase is basically a string type. It is OK to use Non-pointer reference If it is optional. By default, it will be an empty string, i.e., "".

Keep it aligned with `SnapshotPhase`. We can update them together later if necessary.